Pod/Classes/PINAnimatedImageManager.m

```diff
@@ -41,14 +46,23 @@ + (instancetype)sharedManager;
 
 @implementation PINAnimatedImageManager
 
-+ (void)initialize
++ (void)load
 {
   if (self == [PINAnimatedImageManager class]) {
-    static dispatch_once_t startupCleanupOnce;
-    dispatch_once(&startupCleanupOnce, ^{
-      dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
-        [self cleanupFiles];
-      });
+    dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_DEFAULT, 0), ^{
```

Comment:
I guess my last comment is that depending on the device load, you may not be guaranteed that the dispatch queue block will run before the app `DidFinishLaunching`, so now you're not guaranteed that sharedManager will run when you think it will. I'd almost say it's better without the dispatch here.